packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -174,15 +173,9 @@ const MediaReplaceFlow = (
 								} }
 							/>
 						</MediaUploadCheck>
-						{ onSelectURL && <MenuItem
-							icon="admin-links"
-							onClick={ () => ( setShowURLInput( ! showURLInput ) ) }
-							aria-expanded={ showURLInput }
-						>
-							<div> { __( 'Insert from URL' ) } </div>
-						</MenuItem> }
 					</NavigableMenu>
-					{ showURLInput && <div className="block-editor-media-flow__url-input">
+					{ onSelectURL && <div className="block-editor-media-flow__url-input">
+						<span className="block-editor-media-replace-flow__image-url-label">{ __( ' Current media URL:' ) }</span>
```

Comment:
```suggestion
						<span className="block-editor-media-replace-flow__image-url-label">{ __( 'Current media URL:' ) }</span>
```
I think we can remove the space.